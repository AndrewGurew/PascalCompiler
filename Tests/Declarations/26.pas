procedure foo(a:double; c:integer);
begin
end