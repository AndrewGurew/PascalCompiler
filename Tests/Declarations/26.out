Declaration
 ‚é¨foo - Procedure
 Params: a - Double, c - Integer


